pytorch_lightning/utilities/grads.py

```diff
@@ -20,7 +20,7 @@
 from torch.nn import Module
 
 
-def grad_norm(module: Module, norm_type: Union[float, int, str]) -> Dict[str, float]:
+def grad_norm(module: Module, norm_type: Union[float, int, str], folder_separator: str) -> Dict[str, float]:
```

Comment:
```suggestion
def grad_norm(module: Module, norm_type: Union[float, int, str], folder_separator: str = '/') -> Dict[str, float]:
```
Let's define a default here so we don't force changing code for all callers